src/main/scala/is/hail/expr/ir/TypeCheck.scala

```diff
@@ -5,16 +5,25 @@ import is.hail.utils._
 
 object TypeCheck {
   def apply(ir: IR, aggEnv: Option[Env[Type]] = None) {
+    apply(ir, new Env[Type](), aggEnv)
```

Comment:
there's nothing in the `try` block, and the `apply` call should catch/throw the right exception, so I think the try/catch thing can go away here?

Reply:
oops, I totally meant to delete that.